ksql-serde/src/main/java/io/confluent/ksql/serde/json/JsonSchemaFormat.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.serde.json;
+
+import io.confluent.connect.json.JsonSchemaData;
+import io.confluent.kafka.schemaregistry.ParsedSchema;
+import io.confluent.kafka.schemaregistry.json.JsonSchema;
+import io.confluent.ksql.serde.Format;
+import io.confluent.ksql.serde.FormatInfo;
+import io.confluent.ksql.serde.KsqlSerdeFactory;
+import org.apache.kafka.connect.data.Schema;
+
+public class JsonSchemaFormat implements Format {
+
+  public static final String NAME = "JSON_SR";
+
+  private JsonSchemaData jsonData;
+
+  public JsonSchemaFormat() {
+    this.jsonData = new JsonSchemaData();
+  }
+
+  @Override
+  public String name() {
+    return NAME;
+  }
+
+  @Override
+  public boolean supportsSchemaInference() {
+    return true;
+  }
+
```

Comment:
Now both `JsonSchemaformat` and `JsonFormat` say they support schema inference from the Schema Registry.  

Is this still correct, or should `JsonFormat` no longer be saying it supports schema inference?  If it is still the case, isn't this going to be confusing for users?

Reply:
This is correct and it's the intended behavior. They can both support schema inference from schema registry, the difference is how you want to serialize/deserialize the data. I will write docs to explain this in a follow-up PR.

Reply:
naming request: can we please call this something other than "schema inference" ? I suggest "schema lookup" or "schema reference" or similar. Reason: "schema inference" in this kind of system usually means "guessing at or figuring out the schema from just the data itself" (a bit like what was recently added to `print topic`) - and as such this is actually a separate feature from anything to do with the SR. 
Thanks for listening, mini-rant over 😁

Reply:
@blueedgenick - I'll update this in a future PR to avoid code jitter